x11-misc/appmenu-gtk/appmenu-gtk-0.7.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils gnome2-utils
+
+DESCRIPTION="Application menu module for GTK"
+HOMEPAGE="https://github.com/rilian-la-te/vala-panel-appmenu"
+SRC_URI="https://github.com/rilian-la-te/vala-panel-appmenu/releases/download/${PV}/${PN}-module.tar.xz"
+S="${WORKDIR}/${PN}-module"
+
+LICENSE="GPL-2 LGPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ppc64 ~sparc ~x86"
```

Comment:
I don't think you can add many of these more exotic keywords to new packages, unless you're part of the arch teams? 
(@gyakovlev)